mjpeg-view/src/main/java/com/github/niqdev/mjpeg/Mjpeg.java

```diff
@@ -21,8 +21,8 @@
 /**
  * A library wrapper for handle mjpeg streams.
  *
- *     <li><a href="https://bitbucket.org/neuralassembly/simplemjpegview">simplemjpegview</a></li>
```

Comment:
there is a `<ul>` missing

Reply:
Lint show me a warning about this. Here I'm the executor and simply deleted it

Reply:
go ahead, whatever you think it's best.. you could just keep the link without the html tbh

Reply:
It is like it is